src/signals/incident-management/containers/IncidentSplitContainer/components/SplitForm/index.js

```diff
@@ -76,19 +111,17 @@ class SplitForm extends React.Component {
   render() {
     const { incident, attachments, subcategories, priorityList, handleCancel } = this.props;
     return (
-      <div className="split-form">
-        <h1>Splitsen</h1>
-
-        <div className="split-form__disclainer-top">
+      <div>
```

Comment:
An empty `<div>` can be replaced by a `<Fragment>`

Reply:
or <></> :-)